templates/login.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends "layout.html.twig" %}
+
+{% block content %}
```

Comment:
jezeli extendujesz layout i ustawiasz w nim content to powinienes umiescic to co ma byc wewnatrz body a nie caly html 

teraz w outpucie powinienes miec: (co nie jest dobrym pomyslem)

```
<!doctype html>
<html>
    <head>...</head>
    <body>
        <!doctype html>
        <html>
            <head>...</head>
            <body>...</body>
        </html>
    </body>
</html>
```

Reply:
Tylko jak pomijam odwołanie do bootstrapa dodanego w head, to rzecz jasna nie mam jego stylów. Gdzie mam go dodać w takim razie?

Reply:
no w pliku ktory extendujesz czyli layout.html.twig